igvm/hypervisor.py

```diff
@@ -802,18 +797,22 @@ def check_netcat(self, port):
     def kill_netcat(self, port):
         self.run('pkill -f "^/bin/nc.openbsd -l -p {}"'.format(port))
 
-    @contextmanager
-    def netcat_to_device(self, device):
+    def netcat_port(self, device):
```

Comment:
I think it is a good idea to split this into a dedicated method :+1:. 

It might be also worth to make it a underscore method `_netcat_port` to hint users that this is a internal method that should not get used outside of the module.